<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>652</width>
    <height>586</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Shiro's Spell Inspector</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <widget class="QSplitter" name="splitter">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <widget class="QTabWidget" name="tabs">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
         <horstretch>2</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="currentIndex">
        <number>-1</number>
       </property>
       <property name="documentMode">
        <bool>true</bool>
       </property>
       <property name="tabsClosable">
        <bool>true</bool>
       </property>
       <property name="movable">
        <bool>true</bool>
       </property>
      </widget>
      <widget class="QWidget" name="verticalLayoutWidget">
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <widget class="QLineEdit" name="quick_search"/>
        </item>
        <item>
         <widget class="QTableWidget" name="results">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="verticalScrollBarPolicy">
           <enum>Qt::ScrollBarAsNeeded</enum>
          </property>
          <property name="horizontalScrollBarPolicy">
           <enum>Qt::ScrollBarAlwaysOff</enum>
          </property>
          <property name="editTriggers">
           <set>QAbstractItemView::NoEditTriggers</set>
          </property>
          <property name="selectionMode">
           <enum>QAbstractItemView::SingleSelection</enum>
          </property>
          <property name="selectionBehavior">
           <enum>QAbstractItemView::SelectRows</enum>
          </property>
          <property name="sortingEnabled">
           <bool>true</bool>
          </property>
          <property name="rowCount">
           <number>0</number>
          </property>
          <property name="columnCount">
           <number>2</number>
          </property>
          <attribute name="verticalHeaderVisible">
           <bool>false</bool>
          </attribute>
          <column>
           <property name="text">
            <string>Id</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>Name</string>
           </property>
          </column>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="exec_btn">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="text">
           <string>Exec</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="code_btn">
          <property name="text">
           <string>Code</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menu_bar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>652</width>
     <height>21</height>
    </rect>
   </property>
   <property name="defaultUp">
    <bool>false</bool>
   </property>
   <widget class="QMenu" name="menu_file">
    <property name="title">
     <string>File</string>
    </property>
    <widget class="QMenu" name="menu_file_version">
     <property name="title">
      <string>Version</string>
     </property>
     <addaction name="action_vers_1_12_1"/>
     <addaction name="action_vers_2_4_3"/>
     <addaction name="action_vers_3_3_5"/>
    </widget>
    <widget class="QMenu" name="menu_auto_completion">
     <property name="title">
      <string>Code Completion</string>
     </property>
     <addaction name="action_code_always"/>
     <addaction name="action_code_space"/>
     <addaction name="action_code_never"/>
    </widget>
    <addaction name="menu_file_version"/>
    <addaction name="menu_auto_completion"/>
    <addaction name="action_icons"/>
    <addaction name="separator"/>
    <addaction name="action_help"/>
    <addaction name="separator"/>
    <addaction name="action_exit"/>
   </widget>
   <addaction name="menu_file"/>
  </widget>
  <action name="action_exit">
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
  <action name="action_vers_1_12_1">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>1.12.1</string>
   </property>
  </action>
  <action name="action_vers_2_4_3">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>2.4.3</string>
   </property>
  </action>
  <action name="action_vers_3_3_5">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>3.3.5</string>
   </property>
  </action>
  <action name="action_icons">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Wowhead Icons</string>
   </property>
  </action>
  <action name="action_code_always">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Always</string>
   </property>
  </action>
  <action name="action_code_space">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>On ^space</string>
   </property>
  </action>
  <action name="action_code_never">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Never</string>
   </property>
  </action>
  <action name="action_help">
   <property name="text">
    <string>Help</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections>
  <connection>
   <sender>quick_search</sender>
   <signal>returnPressed()</signal>
   <receiver>exec_btn</receiver>
   <slot>click()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>511</x>
     <y>45</y>
    </hint>
    <hint type="destinationlabel">
     <x>506</x>
     <y>537</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
